urltest

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
Shouldn't we remove this until we have a complete working URL test file which will solve #27? 
A different PR can be made for that.